### PR TITLE
[GH-13333] UI Automation: Write an automated test using Cypress for "Individual "User" and "Channel" screens are scrollable

### PR DESCRIPTION
**Pull request by larkox**
#### Summary

UI Automation: Write an automated test using Cypress for "Individual "User" and "Channel" screens are scrollable
#### Ticket Link

https://github.com/mattermost/mattermost-server/issues/13333
#### Related Pull Requests

None

#### Screenshots

None

### CODE REVIEW

**Comment by larkox** [comment]
This PR is pending this bug to be solved: https://mattermost.atlassian.net/browse/MM-14489
I have an open PR trying to solve that bug here: https://github.com/mattermost/mattermost-webapp/pull/4596

File: e2e/cypress/integration/interactive_dialog/scrollable_spec.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
```
Comment (Willyfrog) [inline e2e/cypress/integration/interactive_dialog/scrollable_spec.js]: would this be possible to force with some sort of `before` step? not sure if there is a ping-alike for this. Having a clear message can help people understand quickly why it failed

File: e2e/cypress/integration/interactive_dialog/scrollable_spec.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                cy.wrap($elForm).find('#interactiveDialogCancel').should('be.visible').and('have.text', 'Cancel');
+                cy.wrap($elForm).find('#interactiveDialogSubmit').should('be.visible').and('have.text', userAndChannelDialog.dialog.submit_label);
+            });
+
+            closeInteractiveDialog();
+        });
+    });
+});
+
+function closeInteractiveDialog() {
```
Comment (Willyfrog) [inline e2e/cypress/integration/interactive_dialog/scrollable_spec.js]: why create a function to use only once?

**Comment by saturninoabril** [comment]
Thanks @larkox! Will review once your bug fix on separate PR is merged.

File: e2e/cypress/integration/interactive_dialog/scrollable_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/interactive_dialog/scrollable_spec.js]: We have this pattern on several spec files. Good point on clear message.  We can probably add check in `before` hook, like,
```javascript
cy.request(Cypress.env().webhookBaseUrl).its('status').should('equal', 200);
```

**Comment by srkgupta** [comment]
Thanks @larkox. Waiting for the pending PR to be merged so that this could be tested and approved.

**Comment by saturninoabril** [comment]
Will review once pending PR - https://github.com/mattermost/mattermost-webapp/pull/4596 - is merged. Removing QA review, just ping us once ready.

**Comment by larkox** [comment]
The PR this was waiting for is already merged.

File: e2e/cypress/integration/interactive_dialog/scrollable_spec.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+let createdCommand;
+let userAndChannelDialog;
+
+describe('Interactive Dialog', () => {
+    before(() => {
+        // # Check webhook
+        cy.request(Cypress.env().webhookBaseUrl).then((response) => {
+            expect(response.status === 200, 'This test requires webhook server running. Initiate `npm run start:webhook` to start.');
+        });
```
Comment (saturninoabril) [inline e2e/cypress/integration/interactive_dialog/scrollable_spec.js]: If webhook server is not running, this terminates Cypress process and generates hard to read error.  I opened [PR-5012](https://github.com/mattermost/mattermost-webapp/pull/5012/files#diff-e28127a472a0ddc91f9bf050e02a0556R81) to address this and I suggest to wait for it to merge so you could use `cy. requireWebhookServer()`

Comment (saturninoabril) [reply e2e/cypress/integration/interactive_dialog/scrollable_spec.js]: fyi, said PR has been merged.  Please update and we're good to merge.

**Pull request merged**